hydrolib/core/dflowfm/extold/models.py

```diff
@@ -11,13 +11,107 @@
     ParsableFileModel,
     SerializerConfig,
 )
+from hydrolib.core.dflowfm.common.models import Operand
 from hydrolib.core.dflowfm.extold.parser import Parser
 from hydrolib.core.dflowfm.extold.serializer import Serializer
 from hydrolib.core.dflowfm.polyfile.models import PolyFile
 from hydrolib.core.dflowfm.tim.models import TimModel
 
-
-class TracerQuantity(str, Enum):
+HEADER = """
```

Comment:
Did you copy paste this from somewhere? 😄 
Do you want me to review this header as well? Or, since it is just a header, let the users report any issues with it (if they find any).

Reply:
Yes this one was provided by @veenstrajelmer, but we will probably update it to a more recent one or change it such that it conforms to our own implementation.